cypress/support/fragments/data_import/mapping_profiles/newFieldMappingProfile.js

```diff
@@ -126,7 +126,70 @@ const addVendor = (profile) => {
   ]);
 };
 
+const getDefaultInstanceMappingProfile = (name) => {
+  const defaultInstanceMappingProfile = {
+    profile: {
+      name,
+      incomingRecordType: 'MARC_BIBLIOGRAPHIC',
+      existingRecordType: 'INSTANCE',
+    }
+  };
+  return defaultInstanceMappingProfile;
+};
+const getDefaultHoldingsMappingProfile = (name, permLocation) => {
+  const defaultHoldingsMappingProfile = {
+    profile: {
+      name,
+      incomingRecordType: 'MARC_BIBLIOGRAPHIC',
+      existingRecordType: 'HOLDINGS',
```

Comment:
let's move this string to const variables to separate constants file which is located in the support directory

Reply:
Done